.gitignore

```diff
@@ -8,4 +8,4 @@ __pycache__/
 .DS_Store
 news-please.iml
 
-venv/
+.newspleaseenv/
```

Comment:
why rename the venv?

Reply:
sorry I added this accidentally, will remove it

Reply:
restored it and squashed it